mapquester_backend/example.env

```diff
@@ -14,3 +14,10 @@ DB_PORT=5432
 # Other settings
 TIME_ZONE=UTC
 LANGUAGE_CODE=en-us
+
+#AWS Connectivity
+AWS_ACCESS_KEY_ID=AKIA6IY353J4KX123456
+AWS_SECRET_ACCESS_KEY=NtdP1kb22NA3yD2uYbLsxnKo
+AWS_STORAGE_BUCKET_NAME=mapquester
+AWS_S3_REGION_NAME=us-east-1
+AWS_S3_PRESIGNED_URL_TIME=3600
```

Comment:
Bro don't commit ACCESS KEYS and IDs

Reply:
Check out the new commit where I have left the access keys as blank and code formatting using black has been done to meet code quality.